models/klaviyo__flows.sql

```diff
@@ -0,0 +1,37 @@
+with flow as (
```

Comment:
Will campaign have similar logic? Or are campaigns going to be handled differently?

Reply:
it'll be basically identical!